src/test/java/LoginTests.java

```diff
@@ -8,25 +8,25 @@ public class LoginTests extends BaseTest {
     @Test
     public void LoginTest() throws InterruptedException {
         HomePage homePage = new LoginPage(driver)
-                .enterEmail("demo@class.com")
-                .enterPassword("te$t$tudent")
+                .enterEmail("uraganchic2012@gmail.com")
+                .enterPassword("Testpro1416!")
```

Comment:
Please, remove code redundancies

**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )**

_For example_

```
  @Test
  public void LoginTest() throws InterruptedException {
    HomePage homePage = new LoginPage(driver).login();
    Assert.assertTrue(homePage.isLoggedIn());
  }
}
```